code/implementation/host-stats/src/main/java/io/cattle/platform/host/stats/api/HostStatsOutputFilter.java

```diff
@@ -16,15 +20,38 @@
     @Override
     public Resource filter(ApiRequest request, Object original, Resource converted) {
         boolean add = false;
+        boolean hostStats = false;
+        boolean containerStats = false;
+        boolean project = false;
 
         if (original instanceof Host && DockerHostConstants.KIND_DOCKER.equals(((Host) original).getKind())) {
             add = true;
+            hostStats = true;
+            containerStats = true;
         } else if (original instanceof Instance && InstanceConstants.KIND_CONTAINER.equals(((Instance) original).getKind())) {
+            containerStats = true;
             add = true;
         }
 
+        if (original instanceof Account && AccountConstants.PROJECT_KIND.equals(((Account) original).getKind())) {
```

Comment:
Could you make these one big if/else along with the above if/else. They're mutually exclusive, right?